### DISCUSSION TITLE
Chat does not respond

### DISCUSSION THREAD

**Discussion by Edelmalz** [Genuitec/Copilot4Eclipse-community#55] [Help]
Hi All,

I am using Eclipse on MacOS and since several weeks I am facing the issue that the Copilot chat does not respond. After typing something in the chat it always processes endless without giving any answer.

The code suggestion works fine.

I already tried to solve that issue by re-installing Copilot4Eclipse, re-login and restarting Eclipse with no success.

![Screenshot 2024-11-21 at 11 29 40](https://github.com/user-attachments/assets/f6739bc3-19f9-4b93-b640-490843274060)

Edit: Correct screenshot has been attached.

**Comment by wayneparrott** [comment]
Something to try:
1) open the chat panel
2) select the [+] to create a new conversation
3) enter "hello" as the prompt
- do you get a meaningful response?

If you still are not receiving chat feedback please try creating a new temporary workspace and test chat within it.

Let me know what you find out.

Wayne
Copilot4Eclipse Team

**Reply by Edelmalz** [reply]
Hi Wayne,

thanks for your support.

Your solution proposals did not work but they lead me to the issue and I found a solution.

It was an issue with our company proxy settings. The funny thing is that Copilot never showed me any issues with the connection in my actual workspace. But by using a new workspace Copilot prompted a connection error.

Neverthless, now it works fine again.

Thank you.

**Reply by wayneparrott** [reply]
Thx for the follow up.
W

**Comment by Edelmalz** [comment]
Hey Guys,

Unfortunately, I have to bring that issue up again. It seems that the "solution" was just temporarily and had nothing to do with company proxy settings. 

Today it stopped working again even when I use my internet at home where no proxy settings are needed.

So I am at the beginning again.

**Reply by Edelmalz** [reply]
Ok, I was able to narrow down the issue to my workspace. I created a new workspace for testing purpose and there Copilot works fine without any issue but when switch back to my actual workspace Copilot does not give any response.

Any ideas?

**Reply by wayneparrott** [reply]
Thanks for the additional info about using different workspaces.

The issue could be that the Copilot4Eclipse Chat database has an issue with reads/writes of your chat conversations to a small repo. The db repo is located at `<workspace-folder>/.metadata/.plugins/com.genuitec.copilot4eclipse/chatdata`.  

Try this experiment:
1. stop Eclipse
2. delete the `chatdata` folder
3. restart Eclipse and see if the issue is cleared

Please share the results of this test.

W

**Reply by Edelmalz** [reply]
I have deleted the chatdata folder but this did not solve the issue. Still the same situation. In my actual workspace chat continuesly processing, in the test workspace the chat response within seconds

**Reply by wayneparrott** [reply]
Is there something different about the workspace file locations that could be causing an issue, different disks...

Can you send us your eclipse log file for review from the workspace that Chat is not working properly? You can send it to support@genuitec.com to my attention with a reference to this thread. You can find the log file at <workspace folder>/.metadata/.log

Do this:
1. stop Eclipse
2.  delete the current log file
3. start Eclipse
4. try submitting a chat prompt that hangs
5. wait 30 seconds for chat prompt to hang then stop eclipse
6. send this newest .log file for review

**Reply by Edelmalz** [reply]
Hi Wayne,

I have sent the mail to the mentioned mail address.

Thanks for your support.

**Reply by wayneparrott** [reply]
@Edelmalz I have not received the email you mentioned above. Please consider resending to support@genuitec.com.

**Reply by Edelmalz** [reply]
@wayneparrott Sent again. Don't know why you haven't received the first one. Hop this one reaches you.
